src/pages/Register.vue

```diff
@@ -122,6 +132,9 @@ export default {
       if (this.$v.$error) return
       this.regCustomer(payload)
     },
+    registerFacebook() {
+      this.registerFb(this)
```

Comment:
try Quasar Plugins: https://quasar-framework.org/guide/app-plugins.html#Special-App-Plugin-Boot
to avoid passing "**this**" around
so if we make common.js a Quasar plugin, then "**store**" will be available